tests/unit/PostgrestFilterTest.php

```diff
@@ -27,118 +29,261 @@ public function setup(): void
     public function testEq(): void
     {
         $result = $this->filter->eq('name', 'Algeria');
+        $reflection = new ReflectionClass($result->{'url'});
+        $url = $reflection->getProperty('host');
+        $query = $reflection->getProperty('query');
+        //$params = $reflection->getProperty('parameters');
         assertEquals($this->filter->path, $result->{'path'});
         assertEquals($this->filter->domain, $result->{'domain'});
         assertEquals($this->filter->scheme, $result->{'scheme'});
-        //assertEquals('eq.Algeria', $result->{'url'}->{'query'});
+        assertEquals('gpdefvsxamnscceccczu.supabase.co', $url->getValue($result->{'url'}));
+        //assertEquals('',$query->getValue($result->{'url'}));
```

Comment:
When trying to assert $result->{'url'}->{'query'} I get that  $query is protected and cannot be accessed.